src/components/network/NetworkItem.vue

```diff
@@ -14,12 +14,11 @@
         {{ networkitem.chain_id }}
       </p>
     </div>
-    <div v-if="networkitem.powered" class="powered-div">
-      <PoweredBy
-        :network="networkitem"
-        :is-current-network="isCurrentNetwork"
-      />
-    </div>
+    <PoweredBy
+      :network="networkitem"
+      :is-current-network="isCurrentNetwork"
+      hide-on-mobile
```

Comment:
Where does this class `hide-on-mobile` come from?

Reply:
it's not a class, it's a prop.

https://vuejs.org/v2/guide/components-props.html#Passing-a-Boolean

Reply:
using a conditional class in the `PoweredBy` component